docs/core/tools/cli-msbuild-architecture.md

```diff
@@ -13,7 +13,7 @@ ms.assetid: 7fff0f61-ac23-42f0-9661-72a7240a4456
 
 # High-level overview of changes in the .NET Core tools
 
-This document will describe in high-level the changes that moving from *project.json* to MSBuild and *.csproj* project system bring. It will outline the new way the tooling is layered all-up and which new pieces are available and what is their place in the overall picture. After reading this article, you should have a better understanding of all of the pieces that make up .NET Core tooling after moving to MSBuild and *.csproj*. 
```

Comment:
I agree with @svick that the runtime version here is irrelevant and I'd go with .NET Core SDK as well. My suggestion:
Break that new sentence in two. The topic is a bit more than just the changes from moving away from project.json, so maybe expand the first part and then the second sentence can be when the changes were introduced. 
For the when answer. technically the change happened with Preview 3. So users that were using VS 2015 and older versions of the SDK, were affected by it.